src/drivers/tty/serial/ttys_dvfs.c

```diff
@@ -58,7 +62,7 @@ int ttys_register(const char *name, void *dev_info) {
 	memset(cdev, 0, sizeof(*cdev));
 	memcpy(cdev->name, name, sizeof(cdev->name));
 	cdev->dev_priv = dev_info;
-	cdev->dev_file.f_ops = &ttys_fops;
+	// cdev->dev_file.f_ops = &ttys_fops;
```

Comment:
I think this probably doesn't work. You have to assign something for `open`.

Reply:
This is new FS, that currently not used in x86/qemu build. I'm working on making old one as close as possible to new, so I could combine them in the end. This will be fixed eventually.